diff --git a/dist/DismissableLayer/utils.js b/dist/DismissableLayer/utils.js
index 12df254e366d238e9a390482462f4ee594f9911c..83afe326022cd0856a0ff63a87fd024728c08b0d 100644
--- a/dist/DismissableLayer/utils.js
+++ b/dist/DismissableLayer/utils.js
@@ -9,9 +9,10 @@ function isLayerExist(layerElement, targetElement) {
 	if (!(targetElement instanceof Element)) return false;
 	const targetLayer = targetElement.closest("[data-dismissable-layer]");
 	const mainLayer = layerElement.dataset.dismissableLayer === "" ? layerElement : layerElement.querySelector("[data-dismissable-layer]");
+	if (targetLayer === mainLayer) return true;
+	if (!targetLayer) return false;
 	const nodeList = Array.from(layerElement.ownerDocument.querySelectorAll("[data-dismissable-layer]"));
-	if (targetLayer && (mainLayer === targetLayer || nodeList.indexOf(mainLayer) < nodeList.indexOf(targetLayer))) return true;
-	else return false;
+	return nodeList.indexOf(mainLayer) < nodeList.indexOf(targetLayer);
 }
 /**
 * Listens for `pointerdown` outside a DOM subtree. We use `pointerdown` rather than `pointerup`
